Components/avatar/avatar.css

```diff
@@ -0,0 +1,36 @@
+.avatar{
+    --size:50px;
```

Comment:
try to work with rem instead of fixed px

Reply:
Done